### DISCUSSION TITLE
[React Refresh] Failed to set up the socket connection.

### DISCUSSION THREAD

**Discussion by nasim67reja** [lxieyang/chrome-extension-boilerplate-react#193] [General]
![FireShot Capture 002 - Extensions - extensions](https://github.com/lxieyang/chrome-extension-boilerplate-react/assets/69707467/bd2ca117-f543-43a7-9d15-f27dd8d1269d)
After cloning ,installing and deploying the same code to chrome i am facing this  react refresh error. Please help me to solve this error. is there anything i am missing.?

**Comment by eburnette** [comment]
Did you ever figure this out? I'm having the same problem.
